packages/domain-views/src/garantiesFinancières/dépôtGarantiesFinancières/dépôtGarantiesFinancières.projector.ts

```diff
@@ -52,19 +50,33 @@ export const registerDépôtGarantiesFinancièresProjector = ({
     switch (event.type) {
       case 'GarantiesFinancièresSnapshot-v1':
         if (!event.payload.aggregate.dépôt) {
-          return;
+          break;
         }
 
         if (isNone(dépôtGarantiesFinancières)) {
           const région = await getRégionsProjet(event.payload.identifiantProjet);
           await create<DépôtGarantiesFinancièresReadModel>(dépôtReadModelKey, {
-            ...event.payload.aggregate.dépôt,
+            typeGarantiesFinancières:
+              event.payload.aggregate.dépôt.typeGarantiesFinancières || undefined,
```

Comment:
dans le type de l'event la clé `typeGarantiesFinancières` peut déjà être `undefined`, alors je pense que tu peux juste faire :
```suggestion
              event.payload.aggregate.dépôt.typeGarantiesFinancières,
```

Reply:
Les valeurs dans l'event peuvent être NULL, j'ai donc fait ça pour appliquer undefined dans ce cas, mais je peux peut être essayer de mettre undefined dans les données passées aux events dans la query plutôt ....